site/es/README.md

```diff
@@ -20,7 +20,7 @@ Por favor revisar [El Anuncio](https://groups.google.com/a/tensorflow.org/d/msg/
 Igualmente, p por favor no traduzcan la seccion de `install/`, cualquier archivo `*.yaml`, o archivo `index.md`.
 Revisar [El Anuncio](https://groups.google.com/a/tensorflow.org/forum/#!msg/docs-zh-cn/mhLp-egzNyE/EhGSeIBqAQAJ).
 
-Note: el siguiente directorio
+Nota: el siguiente directorio
```

Comment:
```suggestion
Note: el siguiente directorio
```

Reply:
Please change back. This must be "Note: ..." because it's a style element on the page that uses a callout box.
(Not used in the GitHub README but this is often copy-pasted into the other docs for the site.)

Reply:
Hi, thank you for the feedback about the 'note' , I tried to check the style element in the viewer version but I'm not able to identify any difference, but if you mention that affect the style is ok. Can you confirm me if in this file in the line 13 the same case? and needs to be changed to 'note' ?